samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -140,10 +140,14 @@
   //
   // Planner can determine which skill functions, if any, need to be used to fullfill a user's request.
   // https://learn.microsoft.com/en-us/semantic-kernel/concepts-sk/planner
+  // - SemanticSkillsDirectory is the root directory to start searching recursively for semantic skills.
+  // - RelevancyThreshold will use embeddings to determine if a skill should be included in a plan's list of possible skills.
   //
   "Planner": {
-    "MaxTokens": 1024,
-    "RelevancyThreshold": 0.78
+    "Enabled": true,
+    "SemanticSkillsDirectory": ".",
+    "MaxTokens": 1536,
+    "RelevancyThreshold": 0.77
```

Comment:
0.77 seems to be an arbitrary threshold. Might worth explaining why 0.77 is selected.

Reply:
makes sense. :)